apps/condo/domains/user/schema/CheckUserExistenceService.js

```diff
@@ -0,0 +1,133 @@
+        message: 'Token not found',
+    },
+    DV_VERSION_MISMATCH: {
+        query: 'checkUserExistence',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Wrong value for data version number',
+    },
+    WRONG_SENDER_FORMAT: {
+        query: 'checkUserExistence',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Invalid format of "sender" field value',
+        correctExample: '{ dv: 1, fingerprint: \'example-fingerprint-alphanumeric-value\'}',
+    },
+}
+
+const CheckUserExistenceService = new GQLCustomSchema('CheckUserExistenceService', {
+    types: [
+        {
+            access: true,
+            type: 'input CheckUserExistenceInput { dv: Int!, sender: SenderFieldInput!, confirmActionToken: ID!, userType: UserTypeType! }',
+        },
+        {
+            access: true,
+            type: 'type CheckUserExistenceOutput { userExists: Boolean!, nameIsSet: Boolean!, emailIsSet: Boolean!, phoneIsSet: Boolean!, passwordIsSet: Boolean! }',
+        },
+    ],
+
+    queries: [
+        {
+            access: access.canCheckUserExistence,
+            schema: 'checkUserExistence(data: CheckUserExistenceInput!): CheckUserExistenceOutput',
+            doc: {
+                summary: 'Using an action token with a verified phone number,' +
+                    ' checks whether a user with the specified type (resident, staff, service) is registered.\n' +
+                    'As a result, information about the completion of some important fields (name, email, phone, password) is also returned.',
+                errors: ERRORS,
+            },
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { confirmActionToken, userType } = data
+
+                await checkDayAnonymousRequestLimitCounters(context, 'checkUserExistence', context.req.ip)
+
+                checkDvAndSender(data, ERRORS.DV_VERSION_MISMATCH, ERRORS.WRONG_SENDER_FORMAT, context)
+
+                if (!confirmActionToken) throw new GQLError(ERRORS.TOKEN_NOT_FOUND, context)
+
+                const action = await ConfirmPhoneAction.getOne(context,
+                    {
+                        token: confirmActionToken,
+                        expiresAt_gte: new Date().toISOString(),
+                        completedAt: null,
+                        isPhoneVerified: true,
+                    },
+                )
+                if (!action) throw new GQLError(ERRORS.TOKEN_NOT_FOUND, context)
+
+                const phoneFromAction = normalizePhone(action.phone) || null
+                if (!phoneFromAction) throw new GQLError(ERRORS.INVALID_PHONE_NUMBER, context)
+
+                const users = await find('User', { type: userType, phone: action.phone })
+                if (users.length > 1) throw new GQLError(ERRORS.MULTIPLE_USERS_FOUND, context)
```

Comment:
Is it possible?
<img width="371" alt="image" src="https://github.com/user-attachments/assets/6c9bec5a-f3c7-41e3-92f1-8f26903085c9">

Reply:
Actually it's not possible, I'll remove this check